string type
geometry_msgs/Pose pose
spatial_db/PolygonMesh geometry
